src/main/cpp/vision/definitions/CameraConfig_302.cpp

```diff
@@ -19,26 +19,27 @@
 #include "vision/definitions/CameraConfig_302.h"
 #include "vision/DragonLimelight.h"
 #include "vision/DragonVision.h"
+#include "vision/DragonQuest.h"
 
 void CameraConfig_302::BuildCameraConfig()
 {
-
     DragonLimelight *front = new DragonLimelight(std::string("limelight-front"), // networkTableName
                                                  DRAGON_LIMELIGHT_CAMERA_IDENTIFIER::FRONT_CAMERA,
-                                                 DRAGON_LIMELIGHT_CAMERA_TYPE::LIMELIGHT4, // PIPELINE initialPipeline,
-                                                 DRAGON_LIMELIGHT_CAMERA_USAGE::BOTH,      // PIPELINE initialPipeline,
-                                                 units::length::meter_t(0.224),            // units::length::inch_t mountingXOffset, /// <I> x offset of cam from robot center (forward relative to robot)
-                                                 units::length::meter_t(-0.08255),         // units::length::inch_t mountingYOffset, /// <I> y offset of cam from robot center (left relative to robot)
-                                                 units::length::meter_t(0.802),            // units::length::inch_t mountingZOffset, /// <I> z offset of cam from robot center (up relative to robot)
-                                                 units::angle::degree_t(-37.0),            // units::angle::degree_t pitch,          /// <I> - Pitch of Camera
-                                                 units::angle::degree_t(0),                // units::angle::degree_t yaw,            /// <I> - Yaw of Camera
-                                                 units::angle::degree_t(0),                // units::angle::degree_t roll,           /// <I> - Roll of Camera
-                                                 DRAGON_LIMELIGHT_PIPELINE::APRIL_TAG,     /// <I> enum for starting pipeline
-                                                 DRAGON_LIMELIGHT_LED_MODE::LED_OFF,       // DRAGON_LIMELIGHT_LED_MODE ledMode,
-                                                 DRAGON_LIMELIGHT_CAM_MODE::CAM_VISION     // CAM_MODE camMode,
+                                                 DRAGON_LIMELIGHT_CAMERA_TYPE::LIMELIGHT4,  // PIPELINE initialPipeline,
+                                                 DRAGON_LIMELIGHT_CAMERA_USAGE::APRIL_TAGS, // PIPELINE initialPipeline,
```

Comment:
I think this is a merge issue, probably should be Both. @witcpalek I believe you reviewed the DriveToAlgae PR, please verify

Reply:
Ignore previous message, as mentioned over teams, we need to revert commits back and re merge. There are a lot of things that lost.